client/src/app/site/history/components/history-list/history-list.component.ts

```diff
@@ -31,75 +32,59 @@ import { ViewUser } from 'app/site/users/models/view-user';
     templateUrl: './history-list.component.html',
     styleUrls: ['./history-list.component.scss']
 })
-export class HistoryListComponent extends BaseComponent implements OnInit {
+export class HistoryListComponent extends BaseModelContextComponent implements OnInit {
     /**
      * Subject determine when the custom timestamp subject changes
      */
     public customTimestampChanged: Subject<number> = new Subject<number>();
 
-    public dataSource: MatTableDataSource<History> = new MatTableDataSource<History>();
+    public dataSource: MatTableDataSource<Position> = new MatTableDataSource<Position>();
 
     public pageSizes = [50, 100, 150, 200, 250];
 
     /**
-     * The form for the selection of the model
-     * When more models are supproted, add a "collection"-dropdown
+     * The form for the selection of the motion
+     * When more models are supported, add a "collection"-dropdown
      */
-    public modelSelectForm: FormGroup;
+    public motionSelectForm: FormGroup;
 
     /**
-     * The observer for the selected collection, which is currently hardcoded
-     * to motions.
+     * The observer for the all motions
      */
-    public collectionObserver: BehaviorSubject<BaseViewModel[]>;
+    public motions: BehaviorSubject<BaseViewModel[]>;
```

Comment:
This variable remains exclusively an instance of `BehaviorSubject<ViewMotion[]>`, doesn't it? Then, declare the type of the variable from `BehaviorSubject<BaseViewModel[]>` to `BehaviorSubject<ViewMotion[]>`

Reply:
Why is here a `BehaviorSubject` used?